draft-kazuho-httpbis-priority.md

```diff
@@ -250,17 +252,6 @@ The value is encoded as an sh-integer.  The default value is zero.
 A server SHOULD transmit HTTP responses in the order of their urgency values.
 The lower the value, the higher the precedence.
 
-The following example shows a request for a CSS file with the urgency set to
-`-1`:
-
-~~~ example
-:method = GET
-:scheme = https
-:authority = example.net
-:path = /style.css
-priority = urgency=-1
-~~~
-
```

Comment:
I think having examples here would greatly help readers internalize how it works as they read the text. We can have a back reference to the header definition.